app/views/posts/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<h1>Create a New Post</h1>
+
+<%= form_with(model: [@user, @post], url: user_posts_path(@user), id: 'new_post') do |form| %>
+  <div class="mb-3">
+    <%= form.label :title, class: "form-label" %>
+    <%= form.text_field :title, class: "form-control" %>
+  </div>
+
+  <div class="mb-3">
+    <%= form.label :text, class: "form-label" %>
+    <%= form.text_area :text, class: "form-control" %>
+  </div>
+
+  <div class="text-end">
+    <%= form.submit 'Create Post', class: "btn btn-primary" %>
+  </div>
+<% end %>
+
+<script>
```

Comment:
- [ ] I left a detailed issue in the video link bebolw, kindly work on the subbmission of the post, without that, I cant confirm the comment and the likes
[the link](https://www.loom.com/share/b962a1f15ccf41a98755783d55d48b3c?sid=0074944e-3279-4898-9f6d-5cf84f8f731f)